ocs_ci/helpers/dr_helpers_ui.py

```diff
@@ -158,7 +158,10 @@ def verify_drpolicy_ui(acm_obj, scheduling_interval):
     """
     ocp_version = get_ocp_version()
     acm_loc = locators[ocp_version]["acm_page"]
-    acm_obj.navigate_data_services()
+    if ocp_version >= "4.16":
```

Comment:
Do we need this condition? I think the right approach would be to make changes as per release branches.